sdk/storage/inc/common/shared_request_options.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "context.hpp"
+
+namespace Azure { namespace Storage {
```

Comment:
Since we have `Details` namespace, should we put this `SharedRequestOptions` in details? Because users are not supposed to use it directly.

Reply:
No, because the shared request options should be visible to customers for its members.

Reply:
okay